### ISSUE TITLE
WinAuth does not appear to see trades awaiting confirmation

### ISSUE THREAD

**Issue by allusernamesrtaken** [winauth/winauth#317]
Reposted this from the steam forums as requested:

Problem described in the title. I'm using WinAuth 3.4.22 and while it generates the codes necessary for logging onto steam correctly, it does not recognise any trades going on my account.

Also, I already had activated that mobile authenticator dogshit on my phone, decided it'd be more convenient to have it on computer instead, so I imported the data from my phone as opposed to disabling the authenticator and reenabling it through WinAuth (hopefully I make sense).

TL;DR: Cannot confirm trades on WinAuth, what do?

**Comment by winauth** [comment]
Are you saying the confirmations window is blank saying "you have no trade confirmations"?

**Comment by allusernamesrtaken** [comment]
Exactly

**Comment by winauth** [comment]
But they are appearing on your phone?

Run WinAuth with logging enabled. Add "-l debug" to the end. To do this, create a shortcut to winauth.exe if you don't have one already, right-click, Properties and add `-l debug` after winauth.exe. Run it again, it'll create winauth.log file in the smae folder or in `\Users\<username>\AppData\Roaming\WinAuth`.

Go into your confirmations window again. If you have "remember me" on, turn it off first, so you have to login again.

Email me the log file, winauth@gmail.com

**Comment by allusernamesrtaken** [comment]
Fixed issue myself, sorry for the false alarm.

Error was on my part when copying the data from the two files on my phone; I didn't double check for any errors.

Thank you for the very quick response and help though! :)

**Comment by winauth** [comment]
Ok, no problem.
